src/reporters/ClearTextReporter.ts

```diff
@@ -12,32 +15,34 @@ export default class ClearTextReporter implements Reporter {
 
   onAllMutantsTested(mutantResults: MutantResult[]): void {
     this.writeLine();
-    var mutantsKilled = 0;
-    var mutantsTimedOut = 0;
-    var mutantsUntested = 0;
+    let mutantsKilled = 0;
+    let mutantsTimedOut = 0;
+    let mutantsUntested = 0;
+
+    // use these fn's in order to preserve the 'this` pointer
+    let logDebugFn = (input: string) => log.debug(input); 
+    let writeLineFn = (input: string) => this.writeLine(input);
+
```

Comment:
You need to `route` the input through these stupid helper methods, so that the `this` pointer does not get corrupted.